### PR TITLE
Loaded profile when not cached

### PR DESCRIPTION
**Pull request by TheHollidayInn**

### CODE REVIEW

File: website/client/components/chat/chatMessages.vue [hunk 1]
```diff
@@ -197,8 +197,14 @@ export default {
 
       return false;
     },
-    showMemberModal (memberId) {
-      const profile = this.cachedProfileData[memberId];
+    async showMemberModal (memberId) {
+      let profile = this.cachedProfileData[memberId];
```
Comment (paglias) [inline website/client/components/chat/chatMessages.vue:201]: what if profile is undefined? Like for old messages before the avatar is fetched?

Comment (TheHollidayInn) [reply website/client/components/chat/chatMessages.vue:201]: You can't click the user if it isn't loaded or if msg.userStyles doesn't exist. I can add the check if you want

Comment (TheHollidayInn) [reply website/client/components/chat/chatMessages.vue:201]: https://github.com/TheHollidayInn/habitrpg/blob/c924fa9446293a0c18bab575a6745d6b5d3165f9/website/client/components/chat/chatMessages.vue#L14

**Comment by paglias** [comment]
Ah perfect!

Il 09 Feb 2018 16:39, "Keith Holliday" <notifications@github.com> ha
scritto:

**Pull request merged**